app/models/course.rb

```diff
@@ -493,6 +493,24 @@ class Course < ActiveHash::Base
           ]
         }
       ]
+    }, {
+      id: 13,
+      name: "RUST",
+      title: 'Rust',
+      description: 'This is a Rust event. The focus will be on programming in Rust.',
```

Comment:
Yes! Your proposed changes are a big improvement. I've added a commit with these changes for Rust.

I'd like to go back and make similar improvements the title and description for Clojure. (I helped with ClojureBridge Pittsburgh a few weeks ago.) I'll do that as a separate pull request.

The title and text for Elm is also very similar. I'll see if I can track down those folks and suggest enhancing the text there as well.